README.md

```diff
@@ -14,7 +14,8 @@
 
 > Used Programming Languages (HTML, CSS, JAVASCRIPT, BOOTSTRAP, YAML)
 
-<a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a> <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a> <a href="https://www.w3schools.com/js/"><img src="https://img.icons8.com/color/48/000000/javascript--v1.png"/><a> <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
+<a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a> <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a> <a href="https://www.w3schools.com/js/"><img src="https://img.icons8.com/color/48/000000/javascript--v1.png"/><a> <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a> <a href="https://www.tutorialspoint.com/yaml/index.htm"><img width="44px" top="0" src="https://upload.wikimedia.org/wikipedia/commons/5/5a/Official_YAML_Logo.svg"/><a>
```

Comment:
Better if you can make a PR next time by reducing the size of the logo little bit @Zweak

Reply:
Hey @YoshithaRathnayake , I've made the necessary changes. I've aligned the logo roughly close to the size of other ones.
You can take a look.
As for the size, I thought it would storing the svg locally would do the trick, it consumes only 1 kb.

Reply:
I've made the same commit to the "NEW-CALCULATOR" repository as well, since it looks better that way.

Reply:
Also, sorry for the delay. My wifi died. 😓

Reply:
> Also, sorry for the delay. My wifi died. 😓

Oh